# ROS message for receiving Google Chat message
string event_time
google_chat_ros/Space space
google_chat_ros/Message message
google_chat_ros/User user
